sessionctx/variable/varsutil.go

```diff
@@ -371,7 +371,7 @@ func parseTimeZone(s string) (*time.Location, error) {
 
 	// The value can be given as a string indicating an offset from UTC, such as '+10:00' or '-6:00'.
 	if strings.HasPrefix(s, "+") || strings.HasPrefix(s, "-") {
-		d, err := types.ParseDuration(s[1:], 0)
+		d, err := types.ParseDuration(nil, s[1:], 0)
```

Comment:
Is it possible that PANIC would be raised in `AppendWarning` called by `parseDatetime` if we pass in nil `*stmtctx.StatementContext` here?

Reply:
Yes, that may happen here...... That's a problem. Some utility functions uses `types.ParseDuration` but it's wired too to pass a `*stmtctx.StatementContext` into an utility function...

Reply:
OK